motion_profiles.py

```diff
@@ -0,0 +1,145 @@
+"""Provide motion profiles for smooth and efficient motion"""
+
+import math
+from typing import NamedTuple
+
+
+class MotionProfile(NamedTuple):
+    """Motion profile representing a specific desired move"""
+    """Robot's acceleration - slope of acceleration triangle"""
+    acceleration: float
```

Comment:
How does this Python code work? I get errors each time I load this.

Reply:
I’m on my phone right now so I can’t help very much. What errors are you getting?

Reply:
The typing.NamedTuple module is a standard library component, it will turn them into instance variables on creation. It’s an immutable tuple whose fields can be accessed like instance members.

Reply:
What errors?

Reply:
Perhaps he isn't running python >= 3.5, I believe the named tuple was only added then.